src/app/(auth)/signup/page.tsx

```diff
@@ -71,8 +78,8 @@ export default function SignUp() {
     <>
       {!emailSentCount && (
         <>
-          <SpacerDiv $gap={10}>
-            <H1>Sign Up</H1>
+          <SpacerDiv $gap={0.625}>
```

Comment:
The value of the gap is in px not rem! 
Also, is the error message supposed to be below instead of above the TextInput? 
![Uploading Screen Shot 2024-04-18 at 5.31.58 PM.png…]()

Reply:
talked to kyrene and she said to keep the error message at the top!